routers/api/v1/repo/issue.go

```diff
@@ -107,19 +106,12 @@ func SearchIssues(ctx *context.APIContext) {
 		}
 		log.Trace("Processing next %d repos of %d", len(repos), count)
 		for _, repo := range repos {
-			switch isClosed {
```

Comment:
Why this is not right if it's consistent?

Reply:
consistent?

Reply:
Maybe I should answer the reason why I deleted is it?

The reason I removed it is because it is a counter that cannot include conditions "Issues Options".
I added function of counter that can include that rule at line 171.

Reply:
This PR doesn't need to use this coarse measure because it calculates the count exactly in models/issue.go:1264:func CountIssues(...) ...